feature-libs/checkout/base/styles/components/order-confirmation/_order-confirmation-thank-you-message.scss

```diff
@@ -4,7 +4,8 @@
     padding: var(--cx-padding, 2.5rem);
 
     h2 {
-      font-weight: $font-weight-normal;
+      font-weight: var(--cx-font-weight-bold);
+      font-size: var(--cx-font-size, 1.5rem);
```

Comment:
Changes to existing CSS styles.  Please review the PR for other instances of changes to styles and add migration docs for each.

Reply:
Added